pkg/runtime/dispatcher.go

```diff
@@ -219,6 +219,13 @@ func (m *dispatcher) Check(ctx context.Context, requestBag attribute.Bag) (*adap
 		},
 	)
 	res, _ := cres.(*adapter.CheckResult)
+	if res == nil { // There was nothing to do.
+		res = &adapter.CheckResult{}
+	}
+
+	if glog.V(3) {
```

Comment:
`V(4)` here and elsewhere. I'm planning to go through and consolidate our logging levels, but for now lets keep things `V(2)` for normal and `V(4)` for verbose. I don't see many different levels providing us value.

Reply:
I think there is value in more levels. 
Sometime you want super verbose logs, they could be v(6) even...
I changed these to v4.